src/App.js

```diff
@@ -1,24 +1,99 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {useState} from 'react';
 import './App.css';
 
-function App() {
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+
+import FoodBox from './components/foodbox/FoodBox'
+import AddFood from './components/addfood/AddFood'
+import Search from './components/search/Search'
+
+const App = () => {
+
+  const [foodList, setFoodList] = useState(foods)
+  const [foodListClient, setFoodListClient] = useState(foods)
+  const [showForm, setShowForm] = useState(false)
+  const [todaysFoods, setTodaysFoods] = useState([])
+  const [totalTodaysCalories, setTotalTodaysCalories] = useState([])
+
+  const handleShowForm = () => {
+    setShowForm(true)
+  }
+
+  const handleNewFoodSubmit = (event) => {
+    event.preventDefault()
+
+    let newFoodName = event.target.name.value
+    let newFoodCalories = event.target.calories.value
+    let newFoodImg = event.target.image.value
+
+    let newFood = {
+      name: newFoodName,
+      calories: newFoodCalories,
+      image: newFoodImg
+    }
+
+    setFoodList([...foodList, newFood])
+    setFoodListClient([...foodList, newFood])
+    setShowForm(false)
+
+  }
+
+  const handleSearch = (event) => {
+    let searchText = event.target.value
+
+    let searchedList = foodList.filter((item) => {
+        return item.name.toLowerCase().includes(searchText.toLowerCase())
+    })
+
+    setFoodListClient(searchedList)
+  }
+
+  const handleTodayListAdd = (foodInfo, quantity) => {
+    let todaysFoodName = foodInfo.name
+    let totalCalories = foodInfo.calories * quantity
+
+    let todaysFoodItem = `${quantity} ${todaysFoodName} = ${totalCalories} calories`
```

Comment:
this works but it would make more sense that you pass the information as an object with its properties quantity, name and calories. that was you can arrange the information in the JSX better.